engine/app/views/good_job/shared/_filter.erb

```diff
@@ -47,5 +47,20 @@
         <% end %>
       <% end %>
     </div>
+
+    <div>
+      <%= form_with(url: "", method: :get) do |form| %>
```

Comment:
you could add here `local: true` as this value might differ because it can be configured globally in the app https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-form_with-label-form_with+options

Reply:
Good catch! Thanks!